src/views/catalog/CreateFromInstanceTypes/components/VMDetailsSection/components/SSHKeySection/SSHKeySection.tsx

```diff
@@ -42,6 +42,7 @@ const SSHKeySection: FC<SSHKeySectionProps> = ({
       : selectedSecretName;
     const secretKey = createSecretOpen ? newKeyValue : undefined;
     setSSHSecretCredentials({ sshSecretName: secretName, sshSecretKey: secretKey });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why? please try to add more checks and avoid this comment

Reply:
@metalice This entire section should be replaced by this PR https://github.com/kubevirt-ui/kubevirt-plugin/pull/1116
so this fix should be temporary
@pcbailey can you please make sure this issue is not happening on your PR?

Reply:
ok